website/docs/en-US/space.md

```diff
@@ -279,6 +279,7 @@ Setting this attribute can adjust the alignment of child nodes, the desirable va
 | prefixCls | Prefix for space-items | string | el-space | - |
 | style     | Extra style rules  | string / Array<Object \| String> / Object | - | - |
 | spacer    | Spacer    | string / number / VNode | - | - |
+| spacer-align      | Spacer align | string | center / left / right  | 'center' |
```

Comment:
I didn't see you adding the attribute to in file `useSpace.ts` `defaultProps`.

Reply:
Error to submit Error to submit 😥